src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/PayRatesService.kt

```diff
@@ -97,6 +113,28 @@ class PayRatesService(
     return newPayRates
   }
 
+  /**
+   * See https://dsdmoj.atlassian.net/browse/SDIT-1605
```

Comment:
would be good to put a summary of the issue here so that people don't have to go to JIRA to read it